### PR TITLE
Add ggalluvial package to dockerfile

### PR DESCRIPTION
**Pull request by zzgeng**
Add ggalluvial package to dockerfile

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -30,17 +31,22 @@ RUN ./install_bioc.r \
...
+	ggalluvial \
 	ggpubr \
 	ggthemes \
 	ggstatsplot \
 	ggfortify \
 	ggrepel \
 	GenomicFeatures \
+	GSVA \
 	msigdbr \
+	readr \
```
Comment (jharenza) [inline Dockerfile:43]:
```suggestion
```

File: Dockerfile [hunk 2]
```diff
@@ -30,17 +31,22 @@ RUN ./install_bioc.r \
...
 	reshape2 \
 	R.utils \
 	survival \
-	survminer
+	survminer \
+	tibble \
+	tidyverse
```
Comment (jharenza) [inline Dockerfile:49]:
```suggestion
	survminer
```

**Review by jharenza — CHANGES_REQUESTED** [review]
@JosephDybas since we use the docker image from rocker/tidyverse (see onboarding [doc](https://www.notion.so/d3b/Repository-and-Pull-Request-Guidelines-aa29e3621a3b42988f8e1941abbaadea?pvs=4)), the docker image comes with the basic tidyverse and data table packages, so we wouldn't need to install them separately.

File: Dockerfile (hunk 2 above)
Comment (JosephDybas) [reply Dockerfile]: Good point. Thanks for the suggestion. I revised the Dockerfile. Thanks.

**Comment by jharenza** [comment]
@zzgeng can you let us know when this is pushed for testing?

**Comment by zzgeng** [comment]
Docker image has been pushed.

```
docker pull pgc-images.sbgenomics.com/zhuangzhuanggeng/d3b_hope_analysis:latest
```

**Review by jharenza — APPROVED** [review]
both packages are in the image

**Pull request merged**